app/models/user.js

```diff
@@ -46,5 +46,11 @@ module.exports = (sequelize, DataTypes) => {
     return User.findOne({ where: { email: e } });
   };
 
+  User.getUsers = (page, size) => {
```

Comment:
Its a good practice to assign default values to this params, so if someone doesn't specify which page and size, we still can respond something. For example you can return as default the first page with 10 elements.